extensions/admin-tool-ext-win/src/main.ts

```diff
@@ -178,13 +178,15 @@ async function launchSsmsDialog(action: string, connectionContext: azdata.Object
 	};
 
 	const args = buildSsmsMinCommandArgs(params);
-
-	Telemetry.sendTelemetryEvent('LaunchSsmsDialog',
-		{
-			action: action,
-			nodeType: oeNode ? oeNode.nodeType : 'Server',
-			authType: connectionContext.connectionProfile.authenticationType
-		});
+	TelemetryReporter.createActionEvent(
+		TelemetryViews.SsmsMinDialog,
+		'LaunchSsmsDialog',
+		'',
+		action).withAdditionalProperties(
+			{
+				nodeType: oeNode ? oeNode.nodeType : 'Server'
+			}).withConnectionInfo(connectionContext.connectionProfile)
+		.send();
```

Comment:
Here's an example of an event that's using both the additional properties and connection info properties functions.

Reply:
Do we need to worry about sending user info up? presumably connection info will have server name, user name, database name, and we probably should not be capturing that in telemetry.

Reply:
Yes, we definitely need to be concerned about that. withConnectionInfo pulls out specific fields and only sends those - I'm not sending the entire connection info structure. The fields it sends are :
authenticationType
providerName
serverType
engineType (this doesn't really exist as a concept in ADS today but we will likely want it in the future so I'm adding it now to reduce overhead later on)

The method actually takes in a separate structure - ITelemetryConnectionInfo - that defines these fields. I just purposely named them to align with the same fields in connectionInfo so that we can easily pass that in without needing to create a new object each time.